packages/rsmax-toutiao/src/__tests__/components/Map.test.tsx

```
@@ -0,0 +1,11 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import { Map } from '../../hostComponents';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)